src/NuGet.Core/NuGet.Commands/RestoreCommand.cs

```diff
@@ -332,6 +332,9 @@ public Task<RestoreResult> ExecuteAsync()
                     _request.MaxDegreeOfConcurrency,
                     token);
 
+            // Clear the local cache for newly installed packages
```

Comment:
In the comments, I would say the in-memory cache or the local repository with the in-memory cache